### ISSUE TITLE
Broken link to a GEDCOM format specification

### ISSUE THREAD

**Issue by 62mkv** [nickreynke/python-gedcom#2]
with version 0.2.0 opening a GEDCOM file, that was only today downloaded from myheritage.com, produces error:
```  File "script.py", line 4, in <module>
    gedcom = Gedcom(file_path)
  File "C:\Python3\lib\site-packages\gedcom\__init__.py", line 148, in __init__
    self.__parse(file_path)
  File "C:\Python3\lib\site-packages\gedcom\__init__.py", line 224, in __parse
    last_element = self.__parse_line(line_number, line.decode('utf-8'), last_element)
  File "C:\Python3\lib\site-packages\gedcom\__init__.py", line 263, in __parse_line
    raise SyntaxError(error_message)
SyntaxError: Line `1` of document violates GEDCOM format
See: http://homepages.rootsweb.ancestry.com/~pmcbride/gedcom/55gctoc.htm
```
This link displays `403 Forbidden` when I try to click it

The first line of the file is: 
```
0 HEAD
```

**Comment by 62mkv** [comment]
this is the only working link I was able to find so far: http://www.phpgedview.net/ged551-5.pdf

**Comment by KeithPetro** [comment]
As of now, this project is implemented for GEDCOM specification version 5.5 (not the 5.5.1 "draft", which many may view as the [de facto current specification](https://www.tamurajones.net/GEDCOM551AnnotatedEdition.xhtml)).

A copy of the 5.5 specification is available here: https://chronoplexsoftware.com/gedcomvalidator/gedcom/gedcom-5.5.pdf

**Comment by nickreynke** [comment]
Sorry for the late reply. I just updated and released a new version to output a valid link.

I will have a look on updating the parser to the 5.5.1 specification in the near future.

Thank you guys and have a nice day!
